tests/decap/test_subnet_decap.py

```diff
@@ -0,0 +1,226 @@
+import pytest
+import logging
+import json
+import time
+import random
+from collections import defaultdict
+
+import ptf.packet as packet
+import ptf.testutils as testutils
+from ptf.mask import Mask
+from tests.common.dualtor.dual_tor_utils import rand_selected_interface     # noqa F401
+from tests.common.fixtures.ptfhost_utils import skip_traffic_test           # noqa F401
+from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py       # noqa F401
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t0', 't1')
+]
+
+DECAP_IPINIP_SUBNET_CONFIG_TEMPLATE = "decap/template/decap_ipinip_subnet_config.j2"
+DECAP_IPINIP_SUBNET_CONFIG_JSON = "decap_ipinip_subnet_config.json"
+DECAP_IPINIP_SUBNET_DEL_TEMPLATE = "decap/template/decap_ipinip_subnet_delete.j2"
+DECAP_IPINIP_SUBNET_DEL_JSON = "decap_ipinip_subnet_delete.json"
+
+SUBNET_DECAP_SRC_IP_V4 = "20.20.20.0/24"
+SUBNET_DECAP_SRC_IP_V6 = "fc01::/120"
+OUTER_DST_IP_V4 = "192.168.0.10"
```

Comment:
This IP is already occupied by the mux server on dualtor testbed, better to use a larger one, like `192.168.0.200`

Reply:
Fixed